frontend/src/components/Mint/MintItemContent.tsx

```diff
@@ -33,51 +37,115 @@ const MintItemContent: React.FC<MintItemContentProps> = ({
   const handleClick = () => {};
 
   return (
-    <IonCard
-      style={{ width: "47%", marginRight: "3%" }}
-      className="card-img-swiper "
-      onClick={handleClick}
-    >
-      {" "}
-      <img
-        className="swiper-item-img"
-        alt="NFT Image"
-        style={{ display: loading ? "none" : "block" }}
-        src={ipfsImageUrl}
-        onLoad={() => setLoading(false)}
-        onError={() => setLoading(false)}
-      />
-      {loading ? (
-        <IonSkeletonText
-          className="swiper-item-img-skeleton"
-          animated={true}
-        ></IonSkeletonText>
-      ) : null}
-      <IonCardHeader>
-        <IonCardTitle>
-          {loading ? (
-            <IonSkeletonText animated={true}></IonSkeletonText>
-          ) : (
-            `Creator.eth`
-          )}
-        </IonCardTitle>
-        <IonCardSubtitle>{cutOffTooLongString(name, 20)}</IonCardSubtitle>
-      </IonCardHeader>
-      <IonCardContent>
-        <IonGrid>
-          <IonRow className="ion-justify-content-between ion-align-items-center">
-            <IonCol size="auto">
-              <IonIcon src="/assets/icons/mint-tile.svg"></IonIcon>
-              <IonLabel>80</IonLabel>
-            </IonCol>
-            <IonCol size="auto">
-              <IonIcon src="/assets/icons/people-tile.svg"></IonIcon>
-              <IonLabel>80</IonLabel>
-            </IonCol>
-          </IonRow>
-        </IonGrid>
-      </IonCardContent>
-    </IonCard>
+    <IonContent>
+      <IonGrid>
+        <IonRow className="ion-justify-content-center">
+          <IonCol size="8">
+            <IonCard className="challenge-mint-card" onClick={handleClick}>
+              <img
+                className="challenge-mint-img"
+                alt="NFT Image"
+                style={{ display: loading ? "none" : "block" }}
+                src={ipfsImageUrl}
+                onLoad={() => setLoading(false)}
+                onError={() => setLoading(false)}
+              />
+              {loading ? (
+                <IonSkeletonText
+                  className="challenge-mint-img-skeleton"
+                  animated={true}
+                ></IonSkeletonText>
+              ) : null}
+              <div className="challenge-time-chip challenge-time-chip-ontop">
+                28 mins left
+              </div>
+              <IonCardHeader>
+                <IonCardTitle>
+                  {loading ? (
+                    <IonSkeletonText animated={true}></IonSkeletonText>
+                  ) : (
+                    `Creator.eth`
+                  )}
+                </IonCardTitle>
+                <IonCardSubtitle>
+                  <div className="name">{cutOffTooLongString(name, 20)}</div>
+                  <div className="cost">0.002 ETH</div>
```

Comment:
We can get the cost of the challenge now: challenge.floorPrice

Reply:
oh thanks, on it will be in the next PR